frameworks/cassandra/src/main/java/com/mesosphere/sdk/cassandra/scheduler/Main.java

```diff
@@ -44,12 +48,21 @@ private static SchedulerBuilder createSchedulerBuilder(File yamlSpecFile) throws
     List<String> localSeeds = CassandraSeedUtils
         .getLocalSeeds(rawServiceSpec.getName(), schedulerConfig);
 
-    return DefaultScheduler.newBuilder(
-        DefaultServiceSpec
-            .newGenerator(rawServiceSpec, schedulerConfig, yamlSpecFile.getParentFile())
-            .setAllPodsEnv("LOCAL_SEEDS", Joiner.on(',').join(localSeeds))
-            .build(),
-        schedulerConfig)
+    DefaultServiceSpec.Generator serviceSpecGenerator =
+        DefaultServiceSpec.newGenerator(
+            rawServiceSpec, schedulerConfig, yamlSpecFile.getParentFile())
+            .setAllPodsEnv("LOCAL_SEEDS", Joiner.on(',').join(localSeeds));
+
+    String yamlBase64 = System.getenv(AUTHENTICATION_CUSTOM_YAML_BLOCK_BASE64_ENV);
+    if (yamlBase64 != null && yamlBase64.length() > 0) {
+      String esYamlBlock = new String(
```

Comment:
what does `es` in `esYamlBlock` stand for?

Reply:
Nice catch, its copied from elastic search, I will update it.

Reply:
updated